src/lsptoolshost/dotnetRuntime/dotnetRuntimeExtensionResolver.ts

```diff
@@ -99,7 +99,7 @@ export class DotnetRuntimeExtensionResolver implements IHostExecutableResolver {
             // Enable dump collection
             env.DOTNET_DbgEnableMiniDump = '1';
             // Collect heap dump
```

Comment:
This comment might need updating. Should this also have a comment why we're including module images too?